puzzles/templates/puzzles_table.html

```diff
@@ -180,6 +182,16 @@ <h5 class="modal-title" id="metatitle-{{ p.pk }}">Assign Meta</h5>
         return search_str;
     };
 
+    // Search by the title of the puzzle, and the term "META" if it's a
+    // metapuzzle.
+    $.fn.dataTableExt.ofnSearch['name-input'] = function(value) {
```

Comment:
Hmm, don't really understand the comment above relative to the code below. What's "title" and what's "META" in the code?

Reply:
Datatable implements pure substring search and filtering based on the search token (by default, the raw HTML as string).

Here I am rewriting the search token to be the title of the puzzle, with "meta" appended to it if it's a meta puzzle.

Reply:
I've updated the comment to be a little more clear.

Reply:
I've also changed the jquery to be less magical and more sensible (and added the relevant class tags to the html).

Reply:
Ah, thank you for that. Was confused what those 'a' and '.badge' references were.